tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -2947,6 +2957,74 @@ def _sanitize_stack_args(self, num_bag_folds, num_bag_sets, num_stack_levels, ti
             raise ValueError(f'num_bag_sets must be an integer. (num_bag_sets={num_bag_sets})')
         return num_bag_folds, num_bag_sets, num_stack_levels
 
+    def interpretable_models_summary(self, verbosity=0):
+        '''Summary of fitted interpretable models along with their corresponding complexities
+        '''
+        d = self.fit_summary(verbosity=verbosity)
+        summaries = pd.DataFrame.from_dict(d)
+
+        complexities = []
+        for i in range(summaries.shape[0]):
+            model_name = summaries.index.values[i]
+            agmodel = self._learner.load_trainer().load_model(model_name)
+            imodel = agmodel.model
+            if not isinstance(agmodel, IModelsModel):
+                complexities.append(np.nan)
+            else:
+                complexities.append(imodel.complexity_)
```

Comment:
Instead, store this information in `predictor.info()` by specifying in IModelsModel:

```
    def get_info(self):
        info = super().get_info()
        info['complexity'] = self.model.complexity_
        return info
```

This way you can completely avoid loading any models.

Reply:
added!